views/js/qtiCreator/plugins/menu/preview.js

```diff
@@ -34,6 +34,60 @@ define([
 ], function($, __, pluginFactory, hider, previewerFactory, buttonTpl){
     'use strict';
 
+    let saved = true;
+
+    /**
+     * Handler for preview=
```

Comment:
```suggestion
     * Handler for preview
```

Reply:
And fix eslint error `Missing JSDoc @param "e" declaration.`

Reply:
done